packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -74,51 +70,18 @@ router.post(
   requestLogger,
   asyncHandler(async (req: Request, res: Response) => {
     if (Config.isSandbox()) return res.sendStatus(httpStatus.NOT_IMPLEMENTED);
-    const failGracefully = getFromQueryAsBoolean("failGracefully", req) ?? false;
+    const failGracefully = getFromQueryAsBoolean("failGracefully", req);
     await rebuildCQDirectory(failGracefully);
     return res.sendStatus(httpStatus.OK);
   })
 );
 
-/**
- * POST /internal/carequality/directory/insert
- *
- * Inserts organizations from a Carequality Directory bundle into our database.
- * @param req.file The Carequality Directory to insert, in JSON format; it should include an array
- *    of Organization resources, property `Bundle.entry` from the original CQ directory payload.
- */
-router.post(
-  "/directory/insert",
-  upload.single("file"),
-  asyncHandler(async (req: Request, res: Response) => {
```

Comment:
Not used anymore, used for the initial load of the CQ dir before we had code to sync it.